### ISSUE TITLE
Update Thai translations

### ISSUE THREAD

**Issue by watchakorn-18k** [squidfunk/mkdocs-material#5131]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "th",
  "action.edit": "แก้ไขหน้านี้",
  "action.skip": "ข้ามไปที่เนื้อหา",
  "action.view": "ดูแหล่งที่มาของหน้านี้ ⬅️",
  "announce.dismiss": "อย่าแสดงสิ่งนี้อีก ⬅️",
  "blog.archive": "คลังเก็บเอกสาร ⬅️",
  "blog.categories": "หมวดหมู่ ⬅️",
  "blog.categories.in": "ใย ⬅️",
  "blog.continue": "อ่านต่อไป ⬅️",
  "blog.draft": "ฉบับร่าง ⬅️",
  "blog.index": "กลับไปยังหน้าแรก ⬅️",
  "blog.meta": "คำอธิบายข้อมูล ⬅️",
  "blog.references": "ลิงก์ที่เกี่ยวข้อง ⬅️",
  "clipboard.copy": "คัดลอก",
  "clipboard.copied": "คัดลอกแล้ว",
  "consent.accept": "ยอมรับ ⬅️",
  "consent.manage": "จัดการการตั้งค่า ⬅️",
  "consent.reject": "ปฏิเสธ ⬅️",
  "footer": "ส่วนท้าย ⬅️",
  "footer.next": "ต่อไป",
  "footer.previous": "ก่อนหน้า",
  "header": "หัวข้อ ⬅️",
  "meta.comments": "ความคิดเห็น",
  "meta.source": "แหล่งที่มา",
  "nav": "ตัวนำทาง ⬅️",
  "readtime.one": "อ่าน 1 นาที ⬅️",
  "readtime.other": "อ่าน # นาที ⬅️",
  "rss.created": "ฟีด RSS ⬅️",
  "rss.updated": "ฟีด RSS ของเนื้อหาที่อัปเดต ⬅️",
  "search": "ค้นหา ⬅️",
  "search.placeholder": "ค้นหา",
  "search.share": "แบ่งปัน ⬅️",
  "search.reset": "ล้าง ⬅️",
  "search.result.initializer": "กำลังเริ่มต้นการค้นหา ⬅️",
  "search.result.placeholder": "พิมพ์เพื่อเริ่มค้นหา",
  "search.result.none": "ไม่พบเอกสารที่ตรงกัน",
  "search.result.one": "พบเอกสารที่ตรงกัน",
  "search.result.other": "พบ # เอกสารที่ตรงกัน",
  "search.result.more.one": "อีกหนึ่งในหน้านี้ ⬅️",
  "search.result.more.other": "# เพิ่มเติมในหน้านี้ ⬅️",
  "search.result.term.missing": "ไม่พบ ⬅️",
  "select.language": "เลือกภาษา ⬅️",
  "select.version": "เลือกเวอร์ชัน ⬅️",
  "source": "ไปที่พื้นที่เก็บข้อมูล",
  "source.file.contributors": "ผู้มีส่วนร่วม ⬅️",
  "source.file.date.created": "สร้าง",
  "source.file.date.updated": "สร้าง",
  "tabs": "แท็บ ⬅️",
  "toc": "สารบัญ",
  "top": "กลับไปด้านบนสุด ⬅️"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to my best knowledge.
- [X] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Thanks! Added in 7efb5fe70.

**Comment by squidfunk** [comment]
Released as part of [9.1.1](https://github.com/squidfunk/mkdocs-material/releases/tag/9.1.1).
